charactersheet/charactersheet/models/common/armor.js

```diff
@@ -25,7 +25,12 @@ function Armor() {
     self.armorStealthOptions = ko.observableArray(Fixtures.armor.armorStealthOptions);
 
     self.acLabel = ko.pureComputed(function() {
-        return 'AC ' + self.armorClass();
+        if (self.armorClass().replace(/^\s+|\s+$/g, '')) {
```

Comment:
@Sonictherocketman That's what I wanted to do originally, but phantom.js doesn't recognize the method.